src/AspNet.Security.OAuth.SuperOffice/SuperOfficeAuthenticationConfigurationManager.cs

```diff
@@ -0,0 +1,33 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using System.Threading;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Microsoft.IdentityModel.Tokens;
+
+namespace AspNet.Security.OAuth.SuperOffice
+{
+    /// <summary>
+    /// Represents the base class for loading configuration and obtaining keys for use with Sign in with SuperOffice.
```

Comment:
Is this product called _"Sign in with SuperOffice"_, or is this copy-paste?

Reply:
Based on previous feedback and requirement to use ConfigurationManager, this is obsolete.